core/templates/pages/exploration-editor-page/services/change-list.service.ts

```diff
@@ -88,7 +93,24 @@ export class ChangeListService implements OnInit {
     private explorationDataService: ExplorationDataService,
     private loaderService: LoaderService,
     private loggerService: LoggerService,
-  ) {}
+    private connectionService: ConnectionService,
+  ) {
+    this.connectionService.onInternetStateChange.subscribe(currentState => {
+      this.hasNetworkConnection = currentState.hasNetworkConnection;
+      this.hasInternetAccess = currentState.hasInternetAccess;
+      if (this.hasNetworkConnection && this.hasInternetAccess) {
+        this.status = 'ONLINE';
```

Comment:
Here and else where please use named constants.

Reply:
Done.